dv/parameters/KDV01_CubeOrangePlusDefault.param

```diff
@@ -596,25 +647,25 @@ MNT2_TYPE,0.0
 MOT_BAT_CURR_MAX,0.0
 MOT_BAT_CURR_TC,5.0
 MOT_BAT_IDX,0.0
-MOT_BAT_VOLT_MAX,0.0
-MOT_BAT_VOLT_MIN,0.0
+MOT_BAT_VOLT_MAX,50.4
+MOT_BAT_VOLT_MIN,36.0
 MOT_BOOST_SCALE,0.0
 MOT_HOVER_LEARN,2.0
```

Comment:
MOT_HOVER_LEARN = 2  is Ok for now. It will automatically learn and save the mid stick hover throttle setting when using different take-off weights/payloads